src/silent-callback.html

```diff
@@ -0,0 +1,7 @@
+<script src="./oidc-client-ts.js"></script>
```

Comment:
i do not see oidc-client-ts.js part of this merge request. can you please add or adapt.

Reply:
it is there no ?
https://github.com/jeffschulzusc/sample-angular-oidc-client-ts/blob/fix-1/angular.json
https://github.com/jeffschulzusc/sample-angular-oidc-client-ts/blob/fix-1/src/silent-callback.html

Reply:
ah yes, that is the trick with the angular.json